pkg/projects/resource_project_membership.go

```diff
@@ -0,0 +1,214 @@
+package projects
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type Equatable interface {
```

Comment:
I think you should implement `Identifiable` on all of them

Reply:
`Equatable` is what you call `Identifiable` :)